.github/review/mcp_trial.py

```
@@ -0,0 +1,332 @@
+    require(all(v.get("commit_id") == meta["head"] and v.get("pull_request_review_id") == review["id"] for v in comments), "inline revision mismatch")
+    return {"id": review["id"], "html_url": review["html_url"], "actor": review["user"]["login"], "inline_count": len(comments)}
+
+
+def export_session(binary, session, cwd, env, output, token):
+    require(re.fullmatch(r"ses_[A-Za-z0-9]+", session or ""), "invalid session ID")
+    path = output / (session + ".json")
+    with path.open("w") as stream:
+        result = subprocess.run([str(binary), "export", session], cwd=cwd, env=env,
+                                stdout=stream, stderr=subprocess.PIPE, timeout=60)
+    require(result.returncode == 0 and path.stat().st_size <= MAX_EVENTS, "session export failed or too large")
+    raw = path.read_bytes(); path.write_bytes(raw.replace(token.encode(), b"[REDACTED]"))
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)